cluster-autoscaler.tf

```diff
@@ -1,12 +1,22 @@
+data "external" "cluster_autoscaler_version" {
```

Comment:
Shouldn't this be a variable instead?

Reply:
That wouldn't work. You cannot execute scripts using a variable.

Reply:
Sorry, I meant using a variable instead of executing a script.

Reply:
But, then you'll need to go to the github repo, find the corresponding matching version for the cluster version that has been used, and finally use that value to set the variable. That sounds like quite a bit of work.